MultiWorldServer/MultiWorldServer.csproj

```diff
@@ -63,6 +63,10 @@
       <Project>{9a6e25ca-f58e-4f1b-9187-7142844a7d0a}</Project>
       <Name>RandomizerLib3.0</Name>
     </ProjectReference>
+    <ProjectReference Include="..\RandomizerMod3.0\RandomizerMod3.0.csproj">
```

Comment:
Personally, I would rather not add a dependency on RandomizerMod to the server if we can help it. Preferably, spoiler/tracker log functionality should be moved to RandomizerLib